internal/uvm/vpmem_mapped.go

```diff
@@ -0,0 +1,307 @@
+package uvm
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Microsoft/hcsshim/internal/guestrequest"
+	"github.com/Microsoft/hcsshim/internal/log"
+	"github.com/Microsoft/hcsshim/internal/memory"
+	"github.com/Microsoft/hcsshim/internal/requesttype"
+	hcsschema "github.com/Microsoft/hcsshim/internal/schema2"
+)
+
+const (
+	PageSize                = 0x1000
```

Comment:
Is this page size in the guest?

Reply:
it's on the host. `vpmem` APIs expect mapping offsets to be page aligned.